Makefile

```diff
@@ -0,0 +1,87 @@
+PACKAGE_NAME=pomegranate
+PY27_ENV=py2.7 # The conda env set up for python 2.7
```

Comment:
I forgot that you can't have inline comments with make.  In trying to comment this for others, I introduced a bad bug that causes the env to include a space.  I will replace this patch to get rid of the bad version.